apps/api/src/app/events/events.controller.ts

```diff
@@ -80,10 +80,9 @@ export class EventsController {
   @ApiOperation({
     summary: 'Trigger event',
     description: `
-    Trigger event is the main (and only) way to send notifications to subscribers. 
-    The trigger identifier is used to match the particular workflow associated with it. 
-    Additional information can be passed according the body interface below.
-    `,
+    Trigger event is the main (and only) way to send notifications to subscribers. The trigger identifier is used to match the particular workflow associated with it. Additional information can be passed according the body interface below.
+    
+    To prevent duplicate triggers, you can optionally pass a \`transactionId\` in the request body. If the same \`transactionId\` is used again, the trigger will be ignored. The retention period depends on your billing tier.    `,
```

Comment:
```suggestion
    Trigger event is the main (and only) way to send notifications to subscribers. The trigger identifier is used to match the particular workflow associated with it. Additional information can be passed according the body interface below.
    
    To prevent duplicate triggers, you can optionally pass a **transactionId** in the request body. If the same **transactionId** is used again, the trigger will be ignored. The retention period depends on your billing tier.`
```